### DISCUSSION TITLE
Converting a number from hex to dec

### DISCUSSION THREAD

**Discussion by Omayya01** [dbeaver/dbeaver#14498] [General]
Hi Good People

Please can you assist me. I want to convert a number from hex to dec in dbeaver but not sure how to script it. 

Would really appreciate any ideas/ query examples.

Thank you!

**Comment by LonwoLonwo** [comment]
Nope, sorry, DBeaver doesn't support this type of conversion.

What database do you use? Maybe we can find another solution with DBeaver tools.

**Comment by Omayya01** [comment]
Hi

I was able to find another solution.   :)

Thank you for your response !

Much appreciated.
